en-US/firefox/features/bookmarks.lang

```diff
@@ -18,53 +20,30 @@ Better bookmarks
 Don’t agonize, lovers of bookmarking. Organize with Firefox.
 
 
-;Features
-Features
-
-
+# File means to arrange (that bookmark).
 ;File that bookmark
 File that bookmark
 
 
+# Dial in means to access (the bookmarks).
 ;Get your faves all sorted with the bookmark star icon, which lets you add custom names and folders quickly. Then dial in your bookmarks toolbar to make sure you never lose sight of the links you love.
 Get your faves all sorted with the bookmark star icon, which lets you add custom names and folders quickly. Then dial in your bookmarks toolbar to make sure you never lose sight of the links you love.
 
 
+# Fly here means to have your bookmark go wherever you go.
```

Comment:
Also specify that English plays with the next "on the fly", but it can be ignored if not possible during translation